### PR TITLE
email: course run sent for review

### PR DESCRIPTION
**Pull request by tasawernawaz**
[ECOM-6081](https://openedx.atlassian.net/browse/ECOM-6081)

### CODE REVIEW

**Comment by tasawernawaz** [comment]
@waheedahmed @awais786 @bderusha Please review.

File: course_discovery/apps/publisher/emails.py [hunk 1]
```diff
@@ -172,6 +174,9 @@ def send_course_workflow_email(course, user, subject, txt_template, html_templat
             'recipient_name': recipient_user.full_name or recipient_user.username if recipient_user else '',
             'sender_name': user.full_name or user.username,
             'course_name': course_name if course_name else course.title,
+            'run_number': run_number,
+            'org_name': course.organizations.all().first().name,
```
Comment (waheedahmed) [inline course_discovery/apps/publisher/emails.py:178]: Instead you can use `course.organization_extension.organization.name`.

**Review by waheedahmed — APPROVED** [review]
Just one comment, 👍 once addressed.

**Review by awais786 — APPROVED** [review]

File: course_discovery/apps/publisher/emails.py (hunk 1 above)
Comment (tasawernawaz) [reply course_discovery/apps/publisher/emails.py:178]: This will break many tests, so I am leaving it.

**Pull request merged**